### PR TITLE
Draft donation patch

### PR DESCRIPTION
**Pull request by aminlatifi**
Added guard to only support draft donation on evm chain
Don't match draft donations have been made in last one minute
Return old draft donation on repetitive save

### CODE REVIEW

**Review by jainkrati — APPROVED** [review]

File: src/resolvers/donationResolver.ts [hunk 1]
```diff
@@ -65,6 +65,12 @@ import {
   getDefaultSolanaChainId,
 } from '../services/chains';
 import { markDraftDonationStatusMatched } from '../repositories/draftDonationRepository';
+import {
+  DRAFT_DONATION_STATUS,
+  DraftDonation,
+} from '../entities/draftDonation';
+
+const draftDonationEnabled = process.env.ENABLE_DRAFT_DONATION === 'true';
```
Comment (mohammadranjbarz) [inline src/resolvers/donationResolver.ts:73]: @aminlatifi  Please add ENABLE_DRAFT_DONATION to `.env.example`

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]
Seems good to me, just please add `ENABLE_DRAFT_DONATION` to `.env.example`

File: src/resolvers/donationResolver.ts (hunk 1 above)
Comment (aminlatifi) [reply src/resolvers/donationResolver.ts:73]: It's already there!

Comment (aminlatifi) [reply src/resolvers/donationResolver.ts:73]: It has been added before.

**Pull request merged**